askama_derive/src/input.rs

```diff
@@ -103,6 +103,25 @@ impl TemplateInput<'_> {
         })
     }
 
+    pub(crate) fn fallback<'n>(
```

Comment:
What's the purpose of this function? What does it do that `new()` doesn't (since it takes the same arguments anyway)? Should have comments for this.